src/Configuration/Programmer/Programmer.cpp

```diff
@@ -68,13 +70,19 @@ void programmer_entry(const CFGCommon_ARG* cmdarg) {
     if (subCmd == "list_device") {
       CFG_POST_MSG("<test>      | Device             |  ID        |  IRLen ");
       CFG_POST_MSG("<test> ----- -------------------- ------------ ----------");
-      CFG_POST_MSG("<test> Found  0 Gemini             0x1000AABB   5");
-      CFG_POST_MSG("<test> Found  1 Gemini             0x2000CCDD   5");
+      CFG_POST_MSG("<test> Found  1 Gemini             0x1000AABB   5");
```

Comment:
It's not clear why we still have dummy output messages. Is this going to be cleaned up ?

Reply:
These dummy output messages is when FOEDAG run without setting the `--compiler`. These messages are for demo purpose.